rocAL/rocAL/include/augmentations/geometry_augmentations/node_rotate.h

```diff
@@ -25,26 +25,24 @@ THE SOFTWARE.
 #include "parameter_factory.h"
 #include "parameter_vx.h"
 #include "graph.h"
+#include "rocal_api_types.h"
 
-class RotateNode : public Node
-{
+class RotateNode : public Node {
 public:
     RotateNode(const std::vector<Tensor *> &inputs, const std::vector<Tensor *> &outputs);
     RotateNode() = delete;
-    void init(float angle);
-    void init(FloatParam *angle);
+    void init(float angle, RocalResizeInterpolationType interpolation_type = ROCAL_LINEAR_INTERPOLATION);
+    void init(FloatParam* angle_param, RocalResizeInterpolationType interpolation_type = ROCAL_LINEAR_INTERPOLATION);
     unsigned int get_dst_width() { return _outputs[0]->info().max_shape()[0]; }
     unsigned int get_dst_height() { return _outputs[0]->info().max_shape()[1]; }
     vx_array get_src_width() { return _src_roi_width; }
     vx_array get_src_height() { return _src_roi_height; }
     vx_array get_angle() { return _angle.default_array(); }
-
 protected:
     void create_node() override;
     void update_node() override;
 private:
     ParameterVX<float> _angle;
-    vx_array _dst_roi_width,_dst_roi_height;
```

Comment:
how is the dst_roi handled?

Reply:
We have a _dst_tensor_roi and _src_tensor_roi in the node which will handle the src and dst coordinates